core/arch/arm/plat-k3/main.c

```diff
@@ -65,6 +63,7 @@ static TEE_Result init_ti_sci(void)
 
 	return TEE_SUCCESS;
 }
+
 service_init(init_ti_sci);
```

Comment:
Unfortunately, the coding style is quite inconsistent currently (I looked at how `driver_init()`/`service_init()`/`early_init()`/`boot_final()` are used with `grep -B1 xxx`, and sometimes there is a blank line, sometimes not.
Given that `DECLARE_KEEP_PAGER()` is similar and uses no blank line, perhaps it is the style we should settle on?
I have no strong opinion either way.

Reply:
ah ok. I added the space because check-patch did complain about not having it. 

I tried to mention checkpatch.sh in the commit message as the reason for the extra line but mentioning the tool also raised an error ...so I opted for just adding the line and moving on.